### ISSUE TITLE
Inconsistent results of each training under the same parameter configuration even when seed is fixed.

### ISSUE THREAD

**Issue by fanqinshan0114** [EricGuo5513/momask-codes#44]
Hello!
I want to ask why the model evaluation results of each training under the same parameter configuration are inconsistent. My environment is exactly the same as the environment.yaml file provided, and I have not modified any code. I want to know if this is normal when the random seed is fixed.

**Comment by Murrol** [comment]
Hi, thanks for using our code.

We found that the random seed for train_vq is not set and just fixed it. Please pull the latest code to use it. 

Feel free to create a pull request to contribute to MoMask.

**Comment by fanqinshan0114** [comment]
> Hi, thanks for using our code.
> 
> We found that the random seed for train_vq is not set and just fixed it. Please pull the latest code to use it.
> 
> Feel free to create a pull request to contribute to MoMask.

It seems that the eval_val_loader in train_vq.py should use the validation set instead of the test set?
